src/rgw/rgw_rados.h

```diff
@@ -480,6 +480,7 @@ class RGWRados
   RGWIndexCompletionManager *index_completion_manager{nullptr};
 
   bool use_cache{false};
+  bool readonly{false};
```

Comment:
I guess this makes sense.  I am more strongly preferring the idea of pulling admin into a remote service all the time.